docs/OnnxReleases.md

```diff
@@ -1,53 +1,43 @@
-The ONNX project, going forward, will plan to release roughly on a two month cadence. We follow the [Semver](https://semver.org/) versioning approach and will make decisions as a community on a release by release basis on whether to do a major or minor release.
+The ONNX project, going forward, will plan to release roughly on a four month cadence. We follow the [Semver](https://semver.org/) versioning approach and will make decisions as a community on a release by release basis on whether to do a major or minor release.
 
-Next expected release: May 15th 
+Next expected release: Sept 15th 2020
 
-Release Checklist: 
+The following is the release process for v1.7.0, including a few manual steps that could be automated for next release:
 
 * Install Twine, a utility tool to interact with pypi. Do  - ``pip install twine``
 * Get hold of the username and password for the ‘onnx’ pypi account.
```

Comment:
Can you add more details regarding where to get the creds from?

Reply:
This is provided to the release manager by Prasanth. I don't think we should put individual's name into the doc. So I will use "ONNX admins".

Reply:
Sure I did not mean individual person... I suggest Steering Committee

Reply:
Sure, updated.